exp/addons/controllers/clusterresourceset_controller.go

```diff
@@ -265,8 +265,11 @@ func (r *ClusterResourceSetReconciler) ApplyClusterResourceSet(ctx context.Conte
 
 	// Iterate all resources and apply them to the cluster and update the resource status in the ClusterResourceSetBinding object.
 	for _, resource := range clusterResourceSet.Spec.Resources {
+
+		mode := addonsv1.ClusterResourceSetStrategy(clusterResourceSet.Spec.Strategy)
```

Comment:
Then line 343 can be removed?
```suggestion
		strategy := addonsv1.ClusterResourceSetStrategy(clusterResourceSet.Spec.Strategy)
```